seed/static/seed/js/controllers/inventory_detail_analyses_modal_controller.js

```diff
@@ -81,7 +81,14 @@ angular.module('BE.seed.controller.inventory_detail_analyses_modal', [])
         if (form.$invalid) {
           return;
         }
-        $scope.waiting_for_server = true
+        $scope.waiting_for_server = true;
+        ['benchmark_data', 'savings_target'].forEach((key, index) => {
+          if (key in $scope.new_analysis.configuration) {
+            if (key in $scope.new_analysis.configuration[key]) {
+              $scope.new_analysis.configuration[key] = $scope.new_analysis.configuration[key][key];
+            }
+          }
+        });
```

Comment:
I'm wondering if there;s a way to avoid doing this. Could we tweak the analysis modal so that the ng-model doesn't create these nested values?

Reply:
AFAIK it's how the form is submitting the select fields but I'll look and see.

Reply:
I had to change how the settings were stored on the controller by making them a list of keys instead of a list of objects to fix this.  A search said it wasn't being used anywhere else so I think this is safe.